BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -149,8 +207,15 @@ extension TipsDetailViewController: UITableViewDataSource, UITableViewDelegate {
   }
   
   func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
-    guard Section(rawValue: indexPath.section) == .tips else { return }
-    
+    guard Section(rawValue: indexPath.section) == .tips &&
+      !tipsList.isEmpty &&
+      tipsList[indexPath.row].category == .recurringTip else { return }
+    let publisherID = tipsList[indexPath.row].id
+    state.ledger.removeRecurringTip(publisherId: publisherID)
+    tipsList.remove(at: indexPath.row)
+    if tipsList.isEmpty {
```

Comment:
You don't reload the table if the tipsList is non-empty after removing it? I'm confused as to what happens here

Reply:
Ah sorry, forgot to add else condition. Changing to updating the entire table.
If the list is empty we need the empty cell with text and if there is not recurring tip in the list then remove the next contribution cell from 0 section.